_rules/iframe-not-focusable-has-no-interactive-content-akn7bn.md

```diff
@@ -28,13 +28,17 @@ acknowledgments:
 
 ## Applicability
 
-This rule applies to any non-focusable `iframe` element that has [focusable][] content.
+This rule applies to any `iframe` element that [contains](#akn7bn:contain) at least one element for which all the following are true:
+- the element is [visible][]; and
+- the element is part of the [flattened tabindex-ordered focus navigation scope][] of the `iframe`.
 
-## Expectation
+An element is <dfn id="akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][]. 
+
+The [flattened tabindex-ordered focus navigation scope][] of an `iframe` is essentially the [sequential focus navigation][] order inside its content.
 
-For each test target, the [nested browsing context][] does not [contain](#akn7bn:contain) elements that are [visible][] and part of the [sequential focus navigation][].
+## Expectation
 
-An element is <dfn id="akn7bn:contain">contained</dfn> in a [nested browsing context][] if its [owner document][] is the [container document][] of the [nested browsing context][].
+None of the test target has a negative number as a `tabindex` [attribute value][].
```

Comment:
I'm actually considering whether we should rather request the iframe to be part of the tab-order of the top-level document. That would be a bit more generic for ways other than negative tabindex  to remove stuff from tab-order. But it also seems that currently negative tabindex (on the iframe) is the only way to do so (for tabbable inside an iframe). So that would complicate the rule with technical mumbo-jumbo without really improving it 🤔

Reply:
I think this might be something we can discuss in the next meeting?

Reply:
+1 to keeping as-is, if we are going to complicate things unnecessarily.